cmd/sebak/cmd/genesis.go

```diff
@@ -122,6 +123,30 @@ func makeGenesisBlock(genesisAddress, commonAddress, networkID, balanceStr, stor
 	}
 	defer st.Close()
 
+	created, err := checkExistingAccounts(st, flagNetworkID, genesisKP.Address(), commonKP.Address(), balance)
+	if err != nil {
+		if created {
+			return "--storage", fmt.Errorf("genesis block is already created, but: %v", err)
+		} else {
+			return "--storage", err
+		}
+	} else if created {
```

Comment:
I think `if created` not `else if` is easy to understand. It's not mandatory :)

Reply:
`created` should be checked, because without error created can be `false`, so without error and `created=false`, creating genesis should be run.

Reply:
@spikeekips I mean,
```
if err != nil {
    if created {
        return "--storage", fmt.Errorf("genesis block is already created, but: %v", err)
    } else {
        return "--storage", err
    }
}

if created {
    if b, err := block.GetBlockByHeight(st, common.GenesisBlockHeight); err != nil {
        return "--storage", fmt.Errorf("failed to get genesis block: %v", err)
    } else {
        log.Info("genesis block already created",
            "height", b.Height,
            "round", b.Round,
            "timestamp", b.Timestamp,
            "total-txs", b.TotalTxs,
            "total-ops", b.TotalOps,
            "proposer", b.Proposer,
        )
    }
    return "", nil
}
```